src/types.ts

```diff
@@ -1,4 +1,3 @@
-import { config } from '@api3/airnode-validator';
```

Comment:
This import was causing an import issue, probably due to mismatched Zod or TS dependencies, but I thought we can just remove it since it's only used for EVM address schema imports 🤷